Corona-Warn-App/src/main/res/layout/contact_diary_overview_day_list_item_test_results.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/card_divider"
+        android:background="?android:attr/listDivider"
```

Comment:
is this by design? can we use the same color that we use already ? I think it is called `hairLineColor`

Reply:
I'm using the same colors and styles as the rest of the contact diary day overview components such as risk events or contacts to keep it consistent